content/millennium/r4/individuals/related-person.md

```diff
@@ -26,7 +26,7 @@ The following fields are returned if valued:
 * [Date of birth](https://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.birthDate){:target="_blank"}
 * [Address](https://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.address){:target="_blank"}
 * [Communication (preferred language)](https://hl7.org/fhir/r4/relatedperson-definitions.html#RelatedPerson.communication){:target="_blank"}
-* [Extensions including related person encounter, relationship level, and period](#extensions){:target="_blank"}
+* [Extensions including related person encounter, relationship level, period and relation](#extensions){:target="_blank"}
```

Comment:
Did Alex request us to add this here? I just noticed we didn't add it for period when it was originally added from the looks of it.